var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -40,6 +40,8 @@ class Openmpi(AutotoolsPackage, CudaPackage):
 
     version("main", branch="main", submodules=True)
 
+    version("5.0.0rc9", branch="main", tag="5.0.0rc9", submodules=True)
```

Comment:
I believe you only want the tag here.  You already have a `main` branch version.

```suggestion
    version("5.0.0rc9", tag="5.0.0rc9", submodules=True)
```

Reply:
It needs a branch because otherwise it fails with a messages that branch 5.0.0rc9 does not exist.  However, I can use the `v5.0.x` branch I think.